modules/custom_operations/CMakeLists.txt

```diff
@@ -4,7 +4,10 @@
 
 cmake_minimum_required(VERSION 3.13)
 
-project(openvino_extensions)
+set(PROJECT_VERSION 2023.3.0) # aligned with OpenVINO version
+set(PROJECT_VERSION_PATCH 0)
```

Comment:
looks like we need to use https://cmake.org/cmake/help/latest/variable/PROJECT_VERSION_TWEAK.html#variable:PROJECT_VERSION_TWEAK

Reply:
replaced